tests/utils.py

```diff
@@ -15,6 +15,18 @@
 TEMP_DATA_FOLDER_VARNAME = 'TEST_RANDOM_DATA_FOLDER'
 
 
+def send_signal_by_process_name(name, signal, timeout=1):
+    separator=' '
+    try:
+        output = subprocess.check_output(['pidof', '-S', separator, name], timeout=timeout)
+    except subprocess.CalledProcessError:
+        return
+    output = output.strip().decode('utf-8').split(separator)
+    for pid in output:
+        os.kill(int(pid), signal)
```

Comment:
If we're using subprocess anyway then it may be more simple to call **killall** utility which sends a signal to processes by their name pattern. It's a pretty old BSD utility which should be generally available everywhere.

Reply:
yes but killall is not installed on all systems by default. but pidof is posix utility that available in more OS's. it more useful for local running tests without requirement to install killall on local/docker machine. 
p.s. debian which used in our tests hasn't installed killall by default